cmd/anubis/main.go

```diff
@@ -174,13 +175,6 @@ func main() {
 
 	internal.InitSlog(*slogLevel)
 
-	if *healthcheck {
-		if err := doHealthCheck(); err != nil {
-			log.Fatal(err)
-		}
```

Comment:
Correct me if I'm wrong here. Health check just doesn't work as you try it before metrics (what it checks against) is initialized

Reply:
Okay, would you be fine if I keep this removed then make a PR to add curl to the image with a default HEALTHCHECK instruction?

Reply:
I don't think ko supports that :)

Reply:
> I don't think ko supports that :)

Oh... Womp womp. 


Revert this then? 

If you've had health checks work as you intended before then it's fine, my attempts to use it failed though